cmd/rookflex/cmd/expander.go

```diff
@@ -0,0 +1,68 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/rook/rook/pkg/daemon/ceph/agent/flexvolume"
+	"github.com/spf13/cobra"
+	"os/exec"
+	"strconv"
+)
+
+var (
+	expandFSCmd = &cobra.Command{
+		Use:   "expandfs",
+		Short: "Expands the size of pod filesystem",
+		RunE:  handleExpandFs,
+	}
+)
+
+func init() {
+	RootCmd.AddCommand(expandFSCmd)
+}
+
+func handleExpandFs(cmd *cobra.Command, args []string) error {
+	var opts = &flexvolume.ExpandOptions{}
+
+	err := json.Unmarshal([]byte(args[0]), opts)
+	if err != nil {
+		return fmt.Errorf(
+			"Rook: Could not parse options for expand %s. Got %v",
```

Comment:
Rook tries to keep normal-length error messages on one line, even if that means going over 100 columns. Please change this to be one line so that it doesn't make the code appear more complex.

Reply:
Yeah, thank you. I will do it.